.github/workflows/validate_pull_request.yml

```diff
@@ -11,7 +11,7 @@ jobs:
       - uses: actions/setup-dotnet@v1
         with:
           dotnet-version: |
-            7.0.304
+            7.0.400
```

Comment:
On an unrelated note, I think you can the whole "default variables" feature to share this constant between all the different yaml files you have, so you only have to update it once in future:
https://docs.github.com/en/actions/learn-github-actions/variables#defining-configuration-variables-for-multiple-workflows